netbox/utilities/templates/helpers/utilization_graph.html

```diff
@@ -9,13 +9,18 @@
       aria-valuemin="0"
       aria-valuemax="100"
       aria-valuenow="{{ utilization }}"
+      {% if utilization == 100 %}
+      class="progress-bar"
+      style="width: {{ utilization }}%; background-color: #800080;"
```

Comment:
The bar color needs to be changed by modifying the `bar_class` context var inside the `utilization_graph` template tag. (`bg-secondary` seems like a good choice.) Please avoid hard-coding color values directly in the HTML.